application/config/config.go

```diff
@@ -851,3 +851,17 @@ func (c *Config) IdeVersion() string {
 func (c *Config) IdeName() string {
 	return c.Engine().GetConfiguration().GetString(configuration.INTEGRATION_ENVIRONMENT)
 }
+
+func (c *Config) IsFedramp() bool {
+	u, err := url.Parse(c.SnykApi())
+	if err != nil {
+		return false
+	}
+	hostnameParts := strings.Split(u.Hostname(), ".")
+	if len(hostnameParts) < 3 {
```

Comment:
Is this check fool proof?

Reply:
Actually yes. A fedramp instance would have at least two `.`. Take for example: `https://app.fedramp.snykgov.io`. But anyhoo, it shouuld be `<4` ;)